df.world-data.xml

```diff
@@ -703,18 +703,20 @@
         <int32_t name="year_tick"/>
 
         -- v0.40.14:
-        <stl-vector name='unk_c0' type-name='int16_t' comment="6 bytes allocated seen. Cannot be pointer"/>
-        <stl-vector name='unk_d0' type-name='int16_t' comment="6 bytes allocated seen. Cannot be pointer"/>
-        <stl-vector name='unk_e0' type-name='int16_t' comment="6 bytes allocated seen. Cannot be pointer. Seen on the same element, so c0, d0, and e0 may be connected"/>
-        <stl-vector name='unk_f0' type-name='int32_t' comment="4 bytes allocated seen. Cannot be pointer"/>
-        <stl-vector name='unk_100' type-name='int32_t' comment="4 bytes allocated seen. Cannot be pointer"/>
-        <stl-vector name='unk_110' type-name='int32_t' comment="4 bytes allocated seen. Cannot be pointer. Seen on the same element, so f0, 100, and 110 may be connected"/>
+        <compound name='picked_growths' comment="also includes 'automatically picked' i.e. fallen fruit that becomes item_spatter">
+            <stl-vector name='x' type-name='int16_t' comment="within the MLT"/>
+            <stl-vector name='y' type-name='int16_t' comment="within the MLT"/>
+            <stl-vector name='z' type-name='int16_t' comment="uncertain coordinate system. 100 = surface on my flat embark, 101 in the air, while the normal coordinates don't go that high"/>
```

Comment:
Does it match either of the coordinates on the right edge of the screen?

Reply:
No, it didn't match any of those.